.npmignore

```diff
@@ -1,2 +1,2 @@
-*
 !dist/**
+typings
```

Comment:
I don't think this should be necessary; typings directory shouldn't be there anyway.